src/components/Navbar.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import {Link} from 'react-router-dom';
+import '../styles/default.css';
+
+export function Navbar() {
+    return (
+        <nav>
+            <ul>
```

Comment:
I think bullet points take up too much space. We need as much space for the gallery as possible. Maybe it's better to do something like this?

`[Gallery] [Wallet] [Settings`